profiles/arch/alpha/package.use.mask

```diff
@@ -1,6 +1,10 @@
 # Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+# Marco Scardovi <marco@scardovi.com> (2021-06-28)
```

Comment:
Instead of masking, it's better to drop arches where you've not tested and then file a rekeywording bug after the affected package is in tree. This eases arch testing because tooling is already built around working with an unkeyworded package, and some automated tooling may not know to unmask.

Reply:
You mean drop/rekeyword evolution-data-server?

Reply:
Done